### DISCUSSION TITLE
ASP .NET Core - do I need to call shutdown on app closing

### DISCUSSION THREAD

**Discussion by Bykiev** [quartznet/quartznet#2114] [Q&A] [answered]
Hi, do I need to manually stop scheduler while the app is stopping when `QuartzHostedService` is used? Unfortunately the [docs](https://www.quartz-scheduler.net/documentation/quartz-3.x/packages/aspnet-core-integration.html#a-practical-example-of-the-setup) are not clear for me

**Comment by lahma** [comment, marked as answer]
When host is stopping, scheduler shutdown will be called:

https://github.com/quartznet/quartznet/blob/400cdd37fe467f32040f77228a9f0f979ff86c62/src/Quartz.Extensions.Hosting/QuartzHostedService.cs#L101
